internal/app/api/router.go

```diff
@@ -26,6 +27,16 @@ func (s *Server) initUserRoutes(group fiber.Router) {
 	userRoutes.Post("/register", s.userHandler.Register)
 }
 
+// initUserRoutes initializes user-related routes.
+func (s *Server) initMenuRoutes(group fiber.Router) {
+	menuRoutes := group.Group("/menu")
+	menuRoutes.Post("/add", s.menuHandler.AddMenu)
+	menuRoutes.Post("/delete/batchuserid", s.menuHandler.DeleteMenuBatchByUserID)
+	menuRoutes.Post("/delete", s.menuHandler.DeleteMenuByMenuId)
+	menuRoutes.Post("/update", s.menuHandler.UpdateMenuByMenuID)
+	menuRoutes.Get("/get", s.menuHandler.GetMenuByUserID)
```

Comment:
for get menu you can create route like this **/{user_id}**  no need to specify get on route